packages/frameworks-sveltekit/src/lib/index.ts

```diff
@@ -266,7 +274,7 @@ const actions: AuthAction[] = [
  */
 export function SvelteKitAuth(
   config:
-    | SvelteKitAuthConfig
+    | (SvelteKitAuthConfig & { basePath?: string })
```

Comment:
```suggestion
    | SvelteKitAuthConfig
```

Likely you forgot to run `pnpm dev` or `pnpm build` for `@auth/core`.